docs/core/extensions/channels.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Channels in .NET
+description: Learn how to use System.Threading.Channels in .NET.
+author: IEvangelist
+ms.author: dapine
+ms.date: 08/03/2022
+---
+
+# System.Threading.Channels library
+
+The <xref:System.Threading.Channels?displayProperty=fullName> namespace provides a set of synchronization data structures for passing data between producers and consumers asynchronously. The library targets .NET Standard and works on all .NET implementations.
+
+This library is available in the [System.Threading.Channels](https://www.nuget.org/packages/System.Threading.Channels) NuGet package (or included when targeting .NET Core 2.1+).
+
+## Producer/consumer conceptual programming model
+
+Channels are an implementation of the producer/consumer conceptual programming model. In this programming model, producers asynchronously produce data, and consumers asynchronously consume that data. In other words, this model hands off data from one party to another. Try to think of channels as you would any other common generic collection type, such as a `List<T>`. The primary difference is that this collection manages synchronization and provides various consumption models through factory creation options. These options control the _bounding strategies_ employed by the channel instance and the behavior of the channel when the bound is reached.
+
+## Bounding strategies
+
+Depending on how a `Channel<T>` is created, its reader and writer will behave differently.
+
+To create a channel that specifies a maximum capacity, call <xref:System.Threading.Channels.Channel.CreateBounded%2A?displayProperty=nameWithType>. To create a channel that can be used by any number of readers and writers concurrently, call <xref:System.Threading.Channels.Channel.CreateUnbounded%2A?displayProperty=nameWithType>. Each bounding strategy exposes various creator-defined options, either <xref:System.Threading.Channels.BoundedChannelOptions> or <xref:System.Threading.Channels.UnboundedChannelOptions> respectively.
+
+> [!NOTE]
+> Regardless of the bounding strategy, a channel will always throw a <xref:System.Threading.Channels.ChannelClosedException> when it's used after it's been closed.
+
+### Unbounded channels
+
+To create an unbounded channel, call one of the <xref:System.Threading.Channels.Channel.CreateUnbounded%2A?displayProperty=nameWithType> overloads:
+
+```csharp
+var channel = Channel.CreateUnbounded<T>();
+```
+
+When you create an unbounded channel, by default, the channel can be used by any number of readers and writers concurrently. Alternatively, you can specify non-default behavior when creating an unbounded channel by providing an `UnboundedChannelOptions` instance. The channel's writer (producer) is unbounded and all writes are performed synchronously. For additional examples, see [Unbounded creation patterns](#unbounded-creation-patterns).
```

Comment:
> The channel's writer (producer) is unbounded

I'm not sure what it means for the "writer" to be unbounded.  It's the channel's capacity that's unbounded.